### PR TITLE
Fix the Windows instructions and a couple of broken links

### PR DESCRIPTION
**Pull request by lnicola**
Just like https://github.com/gtk-rs/docs/pull/11, but hopefully on the right file.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
cc @EPashkin (no idea for windows things...)

File: docs-src/requirements.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Contrary to earlier instructions, **you don't need to uncheck "Linker and
 platform libraries" in the Rust setup or delete `gcc.exe` and `ld.exe` in Rust's
 `bin` directory.**
 
-[GNU ABI]: https://www.rust-lang.org/downloads.html#win-foot
+[GNU ABI]: https://github.com/rust-lang-nursery/rustup.rs/blob/master/README.md#working-with-rust-on-windows
```
Comment (EPashkin) [inline docs-src/requirements.md]: IMHO https://www.rust-lang.org/install.html is better, it have your link too.

Comment (lnicola) [reply docs-src/requirements.md]: Are you sure about this? The text specifically mentions the GNU ABI, which isn't mentioned on that page.

File: docs-src/requirements.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ The GTK+ Project [recommends][gtk-download] using the GTK+ SDK provided by [MSYS
 [gtk-download]: http://www.gtk.org/download/windows.php
 [MSYS2]: https://msys2.github.io/
```
[the remark this answers is not in the document]
Comment (lnicola) [reply docs-src/requirements.md]: I changed this, but the PR didn't update yet (GitHub has been acting up lately).

File: docs-src/requirements.md (hunk 1 above)
Comment (EPashkin) [reply docs-src/requirements.md]: Yes, I was wrong

**Comment by EPashkin** [comment]
Thanks, @lnicola Squash it please.

@GuillaumeGomez IMHO it good to merge after squash

**Comment by lnicola** [comment]
Squashed

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**